### DISCUSSION TITLE
Fine-Tuning a Fine-Tuned PaddleOCR Model on Noisy Data?

### DISCUSSION THREAD

**Discussion by ep0p** [PaddlePaddle/PaddleOCR#14015] [Q&A] [answered]
I recently fine-tuned a pre-trained PaddleOCR model on pristine documents, but I overlooked that many of my real-world documents are noisy (e.g., scanned with artifacts).

Is it possible to fine-tune this already fine-tuned model on the new noisy dataset? If so, what are the best practices for doing this (e.g., learning rates, freezing layers)? Also, how can I avoid degrading the model's performance on pristine data?

**Comment by GreatV** [comment, marked as answer]
for fine-tune model please refer to our Chinese version documentation for the most updated information:
- https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_train/finetune.html
- https://paddlepaddle.github.io/PaddleOCR/latest/FAQ.html#211
